example/core_v1alpha1_observatorium.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: core.observatorium.io/v1alpha1
+kind: Observatorium
+metadata:
+  name: observatorium-xyz
```

Comment:
We should make sure this object gets gets the common labels that all other resources have

Reply:
Done